doc/rst/conf.py

```diff
@@ -54,15 +54,17 @@
 # built documents.
 #
 # The short X.Y version.
-# version = '1.15'
+# version = '1.16'
 
 # We use a custom version variable (shortversion) instead, because setting
 # 'version' adds a redundant version number onto the top of the sidebar
 # automatically (rtd-theme). We also don't use |version| anywhere in rst
-shortversion = 1.15
+
+shortversion = "1.16 pre-release".replace('-', '&#8209')
+# Replace `-` with special character `&#8209` to prevent line-breaks in HTML href
```

Comment:
I notice one very minor thing about the non-breaking hyphen char. It shows up in the web page "Title".
```<title>Chapel Quickstart Instructions &mdash; Chapel Documentation 1.16 pre&amp;#8209release</title>```
 In Chrome, hover over the Chrome tab w one of these web pages, you can see the  ```&amp;#8209```

This does not seem very important though.